community/snapcast/APKBUILD

```diff
@@ -0,0 +1,69 @@
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=snapcast
+pkgver=0.14.0
+pkgrel=0
+pkgdesc="Synchronous multi-room audio player"
+url="https://github.com/badaix/snapcast"
+arch="all"
+license="GPL-3.0"
+depends="snapserver snapclient"
+makedepends="bash avahi-dev aixlog popl libvorbis-dev flac-dev asio-dev alsa-lib-dev"
+subpackages="$pkgname-doc snapserver snapclient"
```

Comment:
Is there any particular reason why you renamed the subpackages? I would personally prefer `$pkgname-server` and `$pkgname-client` since this is consistent with other packages, e.g. openssh or mosh.

Reply:
I renamed it because... Well... they're the upstream names. I personally prefer to go with upstream wherever possible, but I guess I can change it back...